lib/blueflood_http_client.js

```diff
@@ -19,6 +19,7 @@
 var url = require('url');
 var http = require('http');
 var request = require('request');
+var Logger = require('log4js').getLogger();
```

Comment:
FYI, if you ever wander into the ele js code it uses [logmagic](https://www.npmjs.com/package/logmagic); however, this [log4js](https://www.npmjs.com/package/log4js) looks much better and more mature/complete. I also like their sense of humor:

> Although it's got a similar name to the Java library log4j, thinking that it will behave the same way will only bring you sorrow and confusion.

In contrast, logmagic starts out with this confidence crushing statement:

> This project is usable. It doesn't do everything it should, yet.